ui/desktop/electron-app/src/helpers/app-updater.js

```diff
@@ -19,6 +21,28 @@ if (debug && process.env.APP_UPDATER_CURRENT_VERSION) {
   currentVersion = process.env.APP_UPDATER_CURRENT_VERSION;
 }
 
+// Returns the real CPU architecture of the machine.
+const returnArchitectureToUpdate = () => {
+  const nodeArchitecture = process.arch;
+
+  if (nodeArchitecture === 'arm64') {
+    return nodeArchitecture;
+  }
+
+  try {
+    const runsOnRosetta = parseInt(
+      execSync('sysctl -in sysctl.proc_translated'),
+    );
+
+    if (runsOnRosetta) {
+      return 'arm64';
+    }
+  } catch (err) {
+    log.error(`returnArchitectureToUpdate error: ${err}`);
```

Comment:
```suggestion
    log.error(`returnArchitectureToUpdate: ${err}`);
```

The error in the message is redundant since we log it as an error

Reply:
Sure